src/main/scala/firrtl_interpreter/Driver.scala

```diff
@@ -66,6 +68,13 @@ trait HasInterpreterOptions {
     }
     .text("seed used for random numbers generated for tests and poison values, default is current time in ms")
 
+  parser.opt[Long]("fint-max-execution-depth")
+    .abbr("fmed")
```

Comment:
I think the abbr for new option should be fimed, the fi prefix to the abbreviation helps to keep all the related interpreter abbreviations distinct